environment.yml

```diff
@@ -2,8 +2,11 @@ name: knoten
 channels:
   - conda-forge
   - usgs-astrogeology
+  - plotly
 dependencies:
-  - bokeh
+  - plotly
```

Comment:
Looks like the rest of the file is in abc order. Could you move the imports to follow the pattern?